src/canvas/SelectableCanvas.ts

```diff
@@ -820,7 +820,8 @@ export class SelectableCanvas<EventSpec extends CanvasEvents = CanvasEvents>
     ) {
       if (
         (this.perPixelTargetFind || obj.perPixelTargetFind) &&
-        !(obj as unknown as IText).isEditing
+        !(obj as unknown as IText).isEditing &&
+        this._activeObject != obj
```

Comment:
We need to look at how this feature has been introduced/built before  thinking is a bug and not a change of feature. I'll dig